package.json

```diff
@@ -39,11 +41,7 @@
     "eslint-plugin-react": "4.2.3",
     "expect": "1.16.0",
     "glob": "7.0.3",
-    "gulp": "3.9.1",
-    "gulp-babel": "6.1.2",
-    "gulp-sourcemaps": "1.6.0",
-    "gulp-vinyl-zip": "1.2.2",
-    "gulp-watch": "4.3.5",
+    "gulp-vinyl-zip": "^1.2.2",
```

Comment:
Only using vynil-zip for packaging. See task-package.js. Should remove it in a followup.